reference/constraints/Unique.rst

```diff
@@ -107,8 +109,8 @@ Options
 
 .. include:: /reference/constraints/_groups-option.rst.inc
 
-message
-~~~~~~~
+``message``
```

Comment:
Is this the case for all constraints? If yes this should be fixed in the lowest maintained branch (4.4) for the constraints.

We would be glad to receive such a PR from your side, but if you don't have time, no worries.

This should not be fixed in this PR, so please revert this line, thanks